syntax = "proto3";

package syft.core.node.domain.service;

import "proto/core/node/domain/service/request_message.proto";
import "proto/core/common/common_object.proto";
import "proto/core/io/address.proto";

message GetAllRequestsMessage {
  syft.core.io.Address address = 1;
  syft.core.io.Address reply_to = 2;
}

message GetAllRequestsResponseMessage {
  syft.core.io.Address address = 1;
  repeated syft.core.node.domain.service.RequestMessage requests = 2;
}
